setup.cfg

```diff
@@ -1,4 +1,4 @@
-[wheel]
+[bdist_wheel]
```

Comment:
What's the difference ?

Reply:
There was a deprecation warning when running `bdist_wheel`.